ReactSkia/core_modules/RSkSpatialNavigator.cpp

```diff
@@ -75,20 +75,10 @@ void RSkSpatialNavigator::updateSpatialNavigatorState(NavigatorStateOperation op
 
     switch(operation) {
         case ComponentAdded:
-#if defined(TARGET_OS_TV) && TARGET_OS_TV
```

Comment:
Is this a empty case now? Can we add Some Debugs or comments here

Reply:
updated with comments.